tests/sentry/utils/audit/tests.py

```diff
@@ -21,6 +21,7 @@ def setUp(self):
         self.org = self.create_organization(owner=self.user)
         self.team = self.create_team(organization=self.org)
         self.project = self.create_project(team=self.team)
```

Comment:
In theory, you should be able to add this into `create_project` like, `create_project(team=self.team, platform='java')`